simplq/src/components/common/Nav/LeftNav.jsx

```diff
@@ -21,9 +21,6 @@ const LeftNav = ({ open, toggleClose }) => {
             How it works
           </a>
         </li>
-        <li>
-          <a>My Queues</a>
-        </li>
         <li>
```

Comment:
Instead of removing this, what do you think about making it go to the homepage where the queues are listed, i.e., redirecting then scrolling the queues section?

Reply:
Done. I have made that in a follow up PR.